src/plays/keeper/CreateArea.jsx

```diff
@@ -43,7 +45,7 @@ function CreateArea(props) {
           value={note.content}
           placeholder="Take a note..."
         />
-        <button disabled={note.title.trim().length === 0 && note.content.trim().length === 0} className={(note.title.trim().length === 0 && note.content.trim().length === 0) ? 'create-button-disabled' : 'create-button'} onClick={submitNote}>< FaPlusCircle /></button>
+        <button disabled={title_len === 0 && content_len === 0} className={(title_len === 0 && content_len === 0) ? 'create-button-disabled' : 'create-button'} onClick={submitNote}>< FaPlusCircle /></button>
```

Comment:
`<button disabled={disableButton} className={disableButton ? 'create-button-disabled' : 'create-button'} onClick={submitNote}>< FaPlusCircle /></button>`

Reply:
do you thhink its good to have this changes?

Reply:
Yes, this is even simpler